packages/Spellcheck-Core.package/SpellingTextParser.class/instance/parseText..st

```diff
@@ -1,25 +1,20 @@
 as yet unclassified
 parseText: aString
```

Comment:
the structure of the function `parseText` is very similar to the structure of `parseCamelCaseText`. Is there a reason why you divided it into two functions?
For example we could write `((self isSeparator: c) or: [c isLetter & c isUppercase]) ifTrue: [...`

Reply:
Yes, there is a reason. Separators work differently that camel case words.